__init__.py

```diff
@@ -443,6 +444,16 @@ def on_gui_page_show(self, message):
         if 'mark-2' not in message.data.get('__from', ''):
             # Some skill other than the handler is showing a page
             self.has_show_page = True
+            
+            # If a skill overrides the animations do not show any
+            override_animations = message.data.get('__animations')
+            if override_animations is True:
```

Comment:
This works but should use equivalence `==` instead of checking identity through `is` (to handle some corner cases)
Maybe the block could be

```python
            if override_animations:
                # Disable animations
                self.log.info('Disabling all animations for page')
                self.override_animations = True
            else:
                self.log.info('Displaying all animations for page')
                self.override_animations = False